megalodon/src/mastodon/stream_listener.ts

```diff
@@ -66,7 +66,9 @@ export default class StreamListener extends EventEmitter implements StreamListen
     let options: AxiosRequestConfig = {
       responseType: 'stream',
       adapter: httpAdapter,
-      cancelToken: this._cancelSource.token
+      cancelToken: this._cancelSource.token,
+      maxContentLength: Infinity,
+      maxBodyLength: Infinity
```

Comment:
Do you need this? Do you ever upload files through StreamListener?

Reply:
No, I simply just found-and-replaced them all.

Is there a reason why it _shouldn't_ be like that?

Reply:
No, I see.